src/styles/main.scss

```diff
@@ -0,0 +1,51 @@
+$days: 31;
+$month-length: (28, 29, 30, 31);
+$start-day: (mon, thu, wed, thu, fri, sat, sun);
```

Comment:
The variables `$days`, `$month-length`, and `$start-day` are defined but not used anywhere in the styles. If they are not required for the current implementation, consider removing them to keep the code clean.